.ci/scripts/run_param_benchmarks.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -eEx
+set -o pipefail
+
+# TODO debug
+exit 0
+
+source /workspace/set-env-dist.sh
+index=$LOCAL_RANK
+export OMPI_COMM_WORLD_SIZE=$WORLD_SIZE
+export OMPI_COMM_WORLD_LOCAL_SIZE=$LOCAL_SIZE
+export OMPI_COMM_WORLD_RANK=$RANK
+export OMPI_COMM_WORLD_LOCAL_RANK=$LOCAL_RANK
+
+if (( $index == 0 )); then
```

Comment:
do we really have so many cores and HCAs on CI node?

Reply:
Removed this script (it's just a placeholder for now).